test_junit_xml.py

```diff
@@ -73,8 +73,8 @@ def test_single_suite_no_test_cases(self):
 
         (ts, tcs) = serialize_and_read(
             TestSuite(
-                'test',
-                [],
+                name='test',
```

Comment:
As you can see after you changed the constructors signature you broke the tests already. That could be someones code :)